packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -433,14 +434,20 @@ export function isValidAttestationData(
   // Otherwise the shuffling is determined by the block at the end of the target epoch
   // minus the shuffling lookahead (usually 2). We call this the "pivot".
   const pivotSlot = computeStartSlotAtEpoch(targetEpoch - 1) - 1;
```

Comment:
Suggest to create a utility function `computeEndSlotAtEpoch` and change the code to below. 

```ts
computeEndSlotAtEpoch(targetEpoch - MIN_SEED_LOOKAHEAD);
```

Also we don't need to go back in pat one extra epoch. We should just go back what that const `MIN_SEED_LOOKAHEAD` defined.

Reply:
This line is out of scope of this PR, would prefer to tackle such change in another PR